### DISCUSSION TITLE
Documentation - Using Windows Subsystem for Linux - mentions two Ubuntu versions

### DISCUSSION THREAD

**Discussion by darenm** [unoplatform/uno#11910] [Q&A]
The WSL documentation at - [Using Windows Subsystem for Linux](https://platform.uno/docs/articles/external/uno.wasm.bootstrap/doc/runtime-execution-modes.html#using-windows-subsystem-for-linux) mentions Ubuntu-18.04 and Ubuntu-20.04 - which is the correct version to use?

> If you have another distribution installed make sure that the 18.04 is the default using wslconfig /s "Ubuntu-20.04". You can list your active distributions with wslconfig /l Note that WSL 2 is considerably slower than WSL 1 for the boostrapper scenario. You will need to set your distribution to version 1 using wsl --set-version "Ubuntu-20.04" 1.

**Comment by jeromelaban** [comment]
Thanks for pointing at the issues. You can use any version of ubuntu, it should not cause much trouble.

**Reply by darenm** [reply]
I ran into a few issues with the `dotnet-setup.sh` script - documented in #11913
